libraries/chain/webassembly/runtimes/eos-vm-oc/compile_trampoline.cpp

```diff
@@ -133,6 +134,15 @@ void run_compile(wrapped_fd&& response_sock, wrapped_fd&& wasm_code, uint64_t st
    std::move(prologue_it, prologue.end(), std::back_inserter(initdata_prep));
    std::move(initial_mem.begin(), initial_mem.end(), std::back_inserter(initdata_prep));
 
+   auto get_resource_size = []() {
+      rusage usage{};
+      getrusage(RUSAGE_SELF, &usage);
+      // ru_maxrss is in kilobytes
+      return usage.ru_maxrss;
+   };
+   tlog("receiver ${a}, wasm size: ${ws} KB, oc code size: ${c} KB, compile resident size: ${rs} MB, time ${t} ms, time since queued ${qt} ms",
```

Comment:
`compile resident size` is a little confusing. On the first glance at the example output, I thought it was something related to the OC compile. It is actually the underlying system resident remory.

Reply:
`compile memory usage` ?

Reply:
I like having something that specifies 'max' as part of it -- "max compile memory usage" maybe